Fingerprints/Fingerprints.txt

```diff
@@ -0,0 +1,10 @@
+1, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
```

Comment:
This file should go to resources, I think. What is it used for?

Reply:
I only create the file in order to be able to read in fingerprints in ART.

Reply:
Do you still need it in this repository, then?

Reply:
If you use it for test cases please put it to test resources